python/lsst/ts/m2gui/utils.py

```diff
@@ -222,3 +224,58 @@ def get_num_actuator_ring(ring):
         return 18
     else:
         raise ValueError(f"Not supported ring: {ring!r}")
+
+
+def prompt_dialog_warning(description, is_prompted=True):
```

Comment:
Please add title argument. Showing dialog without title isn't too user friendly - title gives you hints what is wrong. Either put to title (set with diaglog.setWindowTitle) description, or "command failed" (you should be able to recover that as __name__ parameter of function callback). Or even better, put to message title the message, and to text traceback, collected as:

```python
In [5]: try:
   ...:     a=1/0
   ...: except Exception:
   ...:     import traceback
   ...:     print(traceback.print_exc())
   ...: 
   ...: 
Traceback (most recent call last):
  File "<ipython-input-5-c5bc8490150b>", line 2, in <module>
    a=1/0
ZeroDivisionError: division by zero
None
```

Reply:
I added the `title` in the initial beginning. Then, I realized it might not be very helpful because I will `exec` the message box and block all the EUI. I do not consider to support the `async` in this EUI, which might not be needed (it is not a CSC). Thanks!

Reply:
Well you should still print traceback. Thst can be extremely handy when debugging. Showing what in title and what + traceback in text would be much better.

Reply:
I added the title in the commit: [a980118](https://github.com/lsst-ts/ts_m2gui/pull/10/commits/a98011828c18155b56eda87a64cfe17656878744) as your suggestion. The title is the command name. Thanks!